capstone/src/main/webapp/profile.html

```diff
@@ -52,7 +52,9 @@ <h1 id="ongoing-challenges">Ongoing Challenges</h1>
         <span class="close" onclick="closeModal('edit')">&times;</span>
         <div id="modal-profile-pic"></div>
         <h2 id="modal-name-container"></h2>
-        <form id="edit-profile">
+        <form id="edit-profile" method="POST" enctype="multipart/form-data">
```

Comment:
I tested this out and it works, but I'm sort of confused why? I'm not super familiar with html forms but I thought there had to be an action="\editProfile" or whatever servlet you want it sent to.

Reply:
Forms are a bit different with Blobstore. The action of the form becomes a URL that points to Blobstore. When the form is submitted, the request first goes to Blobstore that takes care of handling the file uploaded by the user and then forwards the request to the servlet (which can get all the other data inputed in the form).

Reply:
Oh interesting, thanks!